lib/api/admin/admin.ts

```diff
@@ -0,0 +1,17 @@
+import { db} from "../../db";
+
+export function getlockers() {
+    return db.prepare("SELECT * FROM lockers").get()
+}
+
+export function getusers() {
+    return db.prepare("SELECT * FROM users").get()
+}
+
+export function getregistrations() {
+    return db.prepare("SELECT * FROM registrations").get()
+}
+
+export function deregisterall(){
+    const statement = db.prepare("DELETE FROM registrations").get()
```

Comment:
I think it makes more sense to use `.run()` here, since you don't need to retrieve any rows.